### PR TITLE
Add legacy CIC tournament articles

### PR DESCRIPTION
**Pull request by ReiFan49**
logo and podium (image) excluded for each tournaments (because either missing, expired, or never existed)

need some crosscheck before merge.

worknote:
- beatmap entry are processed manually and parsed based on one of tournament format (_I had to create a quickscript to fetch those_)
- user data, contestant order, staff data, and such are based from _manually mined_ CSV data, and its modified for efficiency.
- fun fact is optional (or even no right) right? (well i want but feel free to point it out)
- stub flag can be removed if it deemed to be a proper entry right

PS: hmu on discord (esp. indonesian contributors for further ID to EN crosschecks)

### CODE REVIEW

File: wiki/Tournaments/CIC/2012/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:178]:
```suggestion
## Match results
```

File: wiki/Tournaments/CIC/2012/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:183]:
```suggestion
| Group | Saturday, 2012-07-28 |  |  |
```
Don't know if this causes problem.
We usually use this format

```
28 July 2012:

| Group | Player 1 |  |  | Player 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Championship | Player 1 | Player 1's score | Player 2's score | The match link |
```

File: wiki/Tournaments/CIC/2012/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:185]:
```suggestion
| Championship | ![][flag_ID] [nirvashzero](https://osu.ppy.sh/users/1254373) (0) | **![][flag_ID] [Rey-volution](https://osu.ppy.sh/users/883990) (3)** | *missing* |
```
I think it's better to specify it missing with also italicised

File: wiki/Tournaments/CIC/2012/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:306]:
```suggestion
[flag_ID]: /wiki/shared/flag/ID.gif "Indonesia"
```
Reference naming

File: wiki/Tournaments/CIC/2012/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:281]:
```suggestion
### General rules
```
Sentence case thingy

File: wiki/Tournaments/CIC/2012/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:294]:
```suggestion
### Match rules
```
Sentence case thingy

**Review by fajar13k — COMMENTED** [review]
Satu dulu bos

File: wiki/Tournaments/CIC/2012/en.md
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2012/en.md:10]: yes it's 1v1

File: wiki/Tournaments/CIC/2012/en.md
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2012/en.md:56]: well I assume it's the old link right?

File: wiki/Tournaments/CIC/2012/en.md
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2012/en.md:183]: I guess I have to generate a format for 1v1 as it's a generalized one (from CIC13)

File: wiki/Tournaments/CIC/2014/en.md
Comment (ReiFan49) [inline wiki/Tournaments/CIC/2014/en.md:27]: holding this out until confirmation from ursa.

**Comment by TPGPL** [comment]
I added some commits to your PR, use `git pull` to synchronise the changes locally

**Comment by ReiFan49** [comment]
since I think there's no need to further hold it,
I leave the footnote on CIC 2014 one, gotta get the final check before merging, that's it.

File: wiki/Tournaments/CIC/2012/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2012/en.md:45]:
```suggestion
---
```
Preferably deleting all Line Breaks

File: wiki/Tournaments/CIC/2013/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2013/en.md:9]:
```suggestion
The **Catch the Beat Indonesia Cup 2013** (***CIC 2013***) was an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [MOD-](https://osu.ppy.sh/users/1272422). It was the second installment of the Catch the Beat Indonesia Cup.
```

File: wiki/Tournaments/CIC/2014/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:9]:
```suggestion
The **Catch the Beat Indonesia Cup 2014** (***CIC 2014***) is an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [urs4chan](https://osu.ppy.sh/users/321063). It is the third installment of the Catch the Beat Indonesia Cup.
```

File: wiki/Tournaments/CIC/2014/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:16]:
```suggestion
| Knockout Stage | 2014-06-06/2014-06-08 |
```
Works better imo

File: wiki/Tournaments/CIC/2014/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:29]:
```suggestion
*Note: Still looking for confirmation of prize pool.*
```

**Review by fajar13k — COMMENTED** [review]
minors

File: wiki/Tournaments/CIC/2014/en.md
Comment (ReiFan49) [reply wiki/Tournaments/CIC/2014/en.md:16]: wait not sentence? 🤔

File: wiki/Tournaments/CIC/2014/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:9]:
```suggestion
The **Catch the Beat Indonesia Cup 2014** (***CIC 2014***) was an Indonesian 4-players osu!catch tournament hosted by ![][flag_ID] [urs4chan](https://osu.ppy.sh/users/321063). It was the third installment of the Catch the Beat Indonesia Cup.
```

File: wiki/Tournaments/CIC/2014/en.md
Comment (fajar13k) [inline wiki/Tournaments/CIC/2014/en.md:33]:
```suggestion
The Catch the Beat Indonesia Cup 2014 was run by various Indonesian osu!catch community members.
```

**Review by fajar13k — COMMENTED** [review]
Very minor ;;;;;

**Review by TPGPL — APPROVED** [review]

**Pull request merged**
